Form/Type/DateRangeType.php

```diff
@@ -20,15 +20,28 @@
 class DateRangeType extends AbstractType
 {
     /**
-     * @var TranslatorInterface
+     * @var TranslatorInterface|null
+     *
+     * @deprecated translator property is deprecated since version 3.x, to be removed in 4.0
```

Comment:
@core23 As I already mentioned in origmnal PR #322 - in the code, there should be exactly written from which version the code is deprecated. Who  is responsible for changing 3.x before tagging the new version?